examples/EU DST example

```diff
@@ -0,0 +1,52 @@
+/* EU DST example */
+/* version 1: dd 23-10-2024 */
+
+#include <RTClib.h>   // https://github.com/adafruit/rtclib 
+RTC_DS3231 rtc; 
+//RTC_DS1307 rtc;
+
+#define USEDST  true // Use DST (true or false)
+
+DateTime now;
```

Comment:
This should be local to `loop()`.

Reply:
It is also used in the setup part.

Reply:
I cannot see the variable `now` used in `setup()`.

Reply:
I think that is not wise to do. 
1. It is a compiler directive and good practice to declare these directives on the top of the code.
2. If you have more complex date/time functionality, it is handy to have this boolean available in the complete program

Reply:
Boolean? I am talking about the variable `now`. I agree that the macro `USEDST` belongs to the top of the file.